app/src/main/java/com/example/packpals/models/Pal.kt

```diff
@@ -1,7 +1,7 @@
 package com.example.packpals.models
 
 data class Pal(
-    val id: String? = null,
```

Comment:
prob better to leave as val but its ok good enough

Reply:
sgtm